src/__forks__/Relay.js

```diff
@@ -16,6 +16,15 @@
 const RelayDefaultNetworkLayer = require('RelayDefaultNetworkLayer');
 const RelayPublic = require('RelayPublic');
 const RelayStore = require('RelayStore');
+const warning = require('warning');
+
+if (__DEV__) {
+  warning(
+    Promise && Array.prototype.find,
```

Comment:
I thought of doing `global.Promise` first but wasn't sure if it gets transformed correctly in browsers.
I'll go with the first. :+1: